src/Acuminator/Acuminator.Utilities/Roslyn/Constants.cs

```diff
@@ -0,0 +1,641 @@
+			{
+				#region BqlCommandInstance
+				internal static readonly string WhereAnd = "WhereAnd";
+				internal static readonly string WhereNew = "WhereNew";
+				internal static readonly string WhereOr = "WhereOr";
+				internal static readonly string AggregateNew = "AggregateNew";
+				internal static readonly string OrderByNew = "OrderByNew";
+				#endregion
+
+				#region BqlCommandStatic
+				internal static readonly string Compose = "Compose";
+				internal static readonly string CreateInstance = "CreateInstance";
+				internal static readonly string AddJoinConditions = "AddJoinConditions";
+				internal static readonly string AppendJoin = "AppendJoin";
+				internal static readonly string NewJoin = "NewJoin";
+				#endregion
+			}
+			#endregion
+
+
+
+			#region DacExtensionDefaultAttributeFix.cs
+
+			public static class DacExtensionDefaultAttributeFixDelegates
+			{
+				public static readonly string PXUnboundDefaultAttributeName = "PXUnboundDefault";
+				public static readonly string PXPersistingCheck = "PXPersistingCheck";
+				public static readonly string PersistingCheck = "PersistingCheck";
+				public static readonly string PersistingCheckNothing = "Nothing";
+				public static readonly string PXDefault = "PXDefault";
+			}
+
+			#endregion
+
+			public static class PXDefaultAttributeFields
+			{
+				public static readonly string PersistingCheck = "PersistingCheck";
+			}
+
+			public enum PXPersistingCheck
+			{
+				Null = 0,
+				NullOrBlank = 1,
+				Nothing = 2
+			}
+
+			#region BqlContext.cs
+
+			public static readonly string SelectBase5 = "PX.Data.SelectBase`5";
+			public static readonly string SearchBase5 = "PX.Data.SearchBase`5";
+			public static readonly string PXSelectBase = "PX.Data.PXSelectBase";
+			public static readonly string Where2 = "PX.Data.Where2`2";
+			public static readonly string And2 = "PX.Data.And2`2";
+			public static readonly string Or2 = "PX.Data.Or2`2";
+			public static readonly string Aggregate = "PX.Data.Aggregate`1";
+			public static readonly string GroupByBase2 = "PX.Data.GroupByBase`2";
+			public static readonly string BqlPredicateBinaryBase2 = "PX.Data.BqlPredicateBinaryBase`2";
+			public static readonly string IBqlCreator = "PX.Data.IBqlCreator";
+			public static readonly string IBqlSelect = "PX.Data.IBqlSelect";
+			public static readonly string IBqlSearch = "PX.Data.IBqlSearch";
+			public static readonly string IBqlJoin = "PX.Data.IBqlJoin";
+			public static readonly string IBqlOn = "PX.Data.IBqlOn";
+			public static readonly string IBqlWhere = "PX.Data.IBqlWhere";
+			public static readonly string IBqlOrderBy = "PX.Data.IBqlOrderBy";
+			public static readonly string IBqlSortColumn = "PX.Data.IBqlSortColumn";
+			public static readonly string IBqlFunction = "PX.Data.IBqlFunction";
+			public static readonly string IBqlPredicateChain = "PX.Data.IBqlPredicateChain";
+			public static readonly string IBqlTable = "PX.Data.IBqlTable";
+
+			#endregion
+
+
+			#region Template
```

Comment:
Why do you need the TemplateAttributeOrClass  or TemplateDelegates? They are not referenced in code